scripts/ci/roxctl-publish/prepare.sh

```diff
@@ -40,11 +40,18 @@ for platform in Linux Darwin Windows; do
   # See https://issues.redhat.com/browse/ROX-14701.
   # We may later want to add binaries with explicit x86_64 architecture which would be roxctl-amd64[.exe].
   if [[ "${platform}" == "Linux" ]]; then
+    cp "${source_dir}/bin/${platform_lower}_arm64/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-arm64"
+    cp "${source_dir}/bin/${platform_lower}_arm64/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-arm64"
+
     cp "${source_dir}/bin/${platform_lower}_ppc64le/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-ppc64le"
     cp "${source_dir}/bin/${platform_lower}_ppc64le/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-ppc64le"
     cp "${source_dir}/bin/${platform_lower}_s390x/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-s390x"
     cp "${source_dir}/bin/${platform_lower}_s390x/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-s390x"
```

Comment:
Nit, since you started visually separating arches (with blank lines), let's do it consistently, please.
```suggestion
    cp "${source_dir}/bin/${platform_lower}_arm64/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-arm64"
    cp "${source_dir}/bin/${platform_lower}_arm64/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-arm64"

    cp "${source_dir}/bin/${platform_lower}_ppc64le/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-ppc64le"
    cp "${source_dir}/bin/${platform_lower}_ppc64le/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-ppc64le"

    cp "${source_dir}/bin/${platform_lower}_s390x/${roxctl_bin}" "${target_dir}/bin/${platform}/${roxctl_bin}-s390x"
    cp "${source_dir}/bin/${platform_lower}_s390x/${roxctl_bin}" "${target_dir}/bin/${platform_lower}/${roxctl_bin}-s390x"
```